src/features/user/TreeMapper/Import/components/SampleTreeCard.tsx

```diff
@@ -3,13 +3,14 @@ import MaterialTextField from '../../../../common/InputTypes/MaterialTextField';
 import i18next from '../../../../../../i18n';
 import styles from '../Import.module.scss';
 import DeleteIcon from '../../../../../../public/assets/images/icons/manageProjects/Delete';
-import { DatePicker, MuiPickersUtilsProvider } from '@material-ui/pickers';
 import { Controller } from 'react-hook-form';
-import DateFnsUtils from '@date-io/date-fns';
-import { ThemeProvider } from '@material-ui/styles';
 import materialTheme from '../../../../../theme/themeStyles';
 import { localeMapForDate } from '../../../../../utils/language/getLanguageName';
-import { InputAdornment, MenuItem } from '@material-ui/core';
+import { InputAdornment, MenuItem, ThemeProvider } from '@mui/material';
```

Comment:
Menu item - slight difference in font sizes as compared to dev.pp.eco (develop branch)

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r867877084

Reply:
Fixed in #1467